h/services/group.py

```diff
@@ -58,6 +58,30 @@ def fetch_by_groupid(self, groupid):
             .one_or_none()
         )
 
+    def filter(self, **kwargs):
```

Comment:
I did this because I anticipate this method expanding in the future to be able to filter on other properties...maybe too premature?

Reply:
Would we just add more arguments to the method, if we need more arguments in the future? 

That might eventually result in too many arguments, but if we get there I think the solution is probably something like introducing a `Filters` object rather than to use `kwargs`. Or changing the API here in some other way. I know we used it once before but I'm not sure that `kwargs` is the right solution for the too-many-arguments problem. I think we want to pass in a single "value object" instead. Like with services, passing in a `models.Group` instead of passing in name, description, scopes, etc individually.